xscen/biasadjust.py

```diff
@@ -187,7 +189,7 @@ def train(
 def adjust(
     dtrain: xr.Dataset,
     dsim: xr.Dataset,
-    simulation_period: list,
+    simulation_periods: list,
```

Comment:
I don't have an idea on how to not have a breaking change, but I might have an idea on breaking it more :P
Should we juste call it `periods` like everywhere else? same for `reference_period` -> `period` in `train`.

Reply:
Those two are indeed the only instances that are not `period(s)` (and `tas_baseline_period` in `subset_warming_level`, but that's different enough). Since we're gonna break it anyway, I'm not against renaming them.

@aulemahal Thoughts?

Reply:
Good with me!